### PR TITLE
docs: added link to the Learn tutorial in Vault CA integration page

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds a link to the Learn tutorial that is available for the Vault CA integration.

### CODE REVIEW

File: website/content/docs/connect/ca/vault.mdx [hunk 1]
```diff
@@ -14,11 +14,13 @@ The Vault CA provider uses the
 [Vault PKI secrets engine](https://www.vaultproject.io/docs/secrets/pki)
 to generate and sign certificates.
 
--> This page documents the specifics of the Vault CA provider.
+This page documents the specifics of the Vault CA provider.
 Please read the [certificate management overview](/docs/connect/ca)
 page first to understand how Consul manages certificates with configurable
 CA providers.
 
+-> **NOTE**: A Learn [tutorial](https://learn.hashicorp.com/tutorials/consul/vault-pki-consul-connect-ca?in=consul/vault-secure) is available to help you configure Vault as Consul Connect service mesh Certification Authority.
```
Comment (dnephin) [inline website/content/docs/connect/ca/vault.mdx:22]: Is there a word missing here?

```suggestion
-> **NOTE**: A Learn [tutorial](https://learn.hashicorp.com/tutorials/consul/vault-pki-consul-connect-ca?in=consul/vault-secure) is available to help you configure Vault as the Consul Connect service mesh Certification Authority.
```

**Review by dnephin — APPROVED** [review]
LGTM, thanks!

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/500162.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 34e1c2b8d2b9f12bab14a7ecd5c1facd8c6a2b17 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 34e1c2b8d2b9f12bab14a7ecd5c1facd8c6a2b17 onto `release/1.10.x` succeeded!

**Pull request merged**